definitions/v3/pixelados.yml

```diff
@@ -83,6 +83,10 @@ search:
   keywordsfilters:
     - name: re_replace
       args: ["\\.", " "]
+    - name: re_replace
+      args: ["(?i)\\bS0*(\\d+)\\b", "T$1"]
```

Comment:
> Reg replace keywordsfilters to change spanish nomenclature of Season (Txx) to standard (Sxx)

This does the opposite of that?

Reply:
Sorry, it's correct but I wrote bad the description... When some apps like Sonarr search for Season send "Juego de Tronos S01" but in the tracker the name is "Juego de Tronos T1" or "T01"